src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -82,7 +84,8 @@ public Statistics(List<UserComment> userCommentList, int topNComments) {
       BigDecimal nextPoint = UPPER_SCORE.min(tempPoint.add(SCORE_INTERVAL));
       Range currentRange = new Range(tempPoint, nextPoint);
       int currentFrequency = 0;
-      PriorityQueue<UserComment> highMagnitudeComments = new PriorityQueue<>();
+      PriorityQueue<UserComment> highMagnitudeComments =
```

Comment:
To make the logic a little clearer, maybe this can be named to `descendingCommentMagnitudeQueue`?